plutus-core/plutus-core/src/PlutusCore/Data.hs

```diff
@@ -16,22 +16,26 @@ import qualified Codec.CBOR.Magic          as CBOR
 import           Codec.Serialise           (Serialise (decode, encode))
 import           Codec.Serialise.Decoding  (decodeSequenceLenIndef, decodeSequenceLenN)
 import           Control.DeepSeq           (NFData)
+import           Control.Lens              ((&), (.~), (?~))
 import           Control.Monad.Except
 import           Data.Bits                 (shiftR)
 import qualified Data.ByteString           as BS
 import qualified Data.ByteString.Lazy      as BSL
+import           Data.Proxy                (Proxy (..))
 import           Data.Text.Prettyprint.Doc
 import           Data.Word                 (Word64, Word8)
+import           GHC.Exts                  (IsList (..))
 import           GHC.Generics
 import           Prelude
 
```

Comment:
There's a blank line between imports. Let's remove it. 

Can you also check the rest of files for this?